m0t1fs/linux_kernel/file.c

```diff
@@ -3114,13 +3117,17 @@ static int pargrp_iomap_dgmode_recover(struct pargrp_iomap *map)
 				db_buf.b_addr;
 			parity[col].b_nob  = PAGE_SIZE;
 		}
-		m0_parity_math_recover(parity_math(map->pi_ioreq), data,
-				       parity, &failed, M0_LA_INVERSE);
+		rc = m0_parity_math_recover(parity_math(map->pi_ioreq), data,
```

Comment:
The same for the return type change.

Reply:
I have kept return type as int for this function because Intel ISA API `gf_invert_matrix()` might return non-zero value if it fails to invert the input matrix. Shall we assert in this case?